### PR TITLE
Gdr 1933

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Changed order of untreated tags
Related JIRA issue: GDR-19333

## Why was it changed?
To unify untreated tags and use `vehicle` as a default

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/fit_curves.R [hunk 1]
```diff
@@ -156,6 +164,9 @@ fit_curves <- function(df_,
 #' @param n_point_cutoff integer indicating number of unique concentrations required to fit curve.
 #' @param capping_fold Integer value of the fold number to use for capping IC50/GR50. Default is \code{5}.
 #'
+#' @examples
+#' logisticFit(0.1, NA)
```
Comment (DominikZabinski) [inline R/fit_curves.R:168]: I get the warning `In value[[3L]](cond) : not enough data points (0 < 4) to perform fitting`. Could you confirm or deny?

File: R/merge_SE.R [hunk 2]
```diff
@@ -7,6 +7,10 @@
 #' @param discard_keys character vector of string that will be discarded
 #' during creating BumpyMatrix object
 #'
+#' @examples
+#' se1 <- get_synthetic_data("finalMAE_small.RDS")[[1]]
+#' merge_SE(list(se1 = se1, se2 = se1))
```
Comment (DominikZabinski) [inline R/merge_SE.R:12]: I get warnings like that: `In .clean_key_inputs(nested_keys, colnames(df_)) :  ignoring input keys: 'fit_source, Metrics_rownames' which are not present in data.table`. Could you confirm or deny?

Comment (bczech) [reply R/merge_SE.R:12]: It is expected

File: R/fit_curves.R (hunk 1 above)
Comment (bczech) [reply R/fit_curves.R:168]: I confirm. We use random data here just to show the usage

**Review by DominikZabinski — APPROVED** [review]

**Pull request merged**